### PR TITLE
Try refactoring out localhost on analytics_api

### PR DESCRIPTION
**Pull request by maxrothman**
@e0d

### CODE REVIEW

**Comment by e0d** [comment]
:+1:

File: playbooks/roles/common_vars/defaults/main.yml [hunk 1]
```diff
@@ -3,6 +3,9 @@
 # to change the base directory
 # where edX is installed
 
+#In docker, set to .local.edx.org (NOTICE LEADING .)
+COMMON_DOMAIN: localhost
```
Comment (fredsmith) [inline playbooks/roles/common_vars/defaults/main.yml:7]: should be ''

**Comment by fredsmith** [comment]
:+1:

**Comment by jibsheet** [comment]
:+1:

**Pull request merged**